reference/OpenAPI.json

```diff
@@ -396,8 +396,73 @@
           }
         ]
       },
-      "Order": {
-        "title": "Order",
+      "OrderRequest": {
```

Comment:
Tu nie powinno być jeszcze Redemption? Bo widzę, że jest tam używany OrderRedemptionDTO

Reply:
Nie :) ja opisuje OrderDTO. Owszem, można się pokusić jeszcze o dodanie później `OrderRedemptionResponse`

Reply:
To z tego co widzę to w OrderDTO jest na samym końcu 
this.redemptions = ...
Ale może to zostać dodane później chyba.

Reply:
Masz racje :) dodamy w czwartek